eng/pipelines/templates/jobs/release.yml

```diff
@@ -1,13 +1,7 @@
 parameters:
-- name: Platforms
+- name: VsixTargets
   type: object
-  default:
-  - linux_x64
-  - linux_arm64
-  - osx_x64
-  - osx_arm64
-  - win_x64
-  - win_arm64
+  default: []
```

Comment:
There shouldn't be a default here.   The caller should always supply an array, no?

Reply:
There could be a case where we don't want to publish vsix packages. Either way it is minor to keep it empty or not existent.